package_picks/package-pick-8.19~2024.01+beta1.sh

```diff
@@ -206,16 +205,16 @@ then
   then
     case "$COQ_PLATFORM_FIATCRYPTO" in
       [yY])
-        PACKAGES="${PACKAGES} coq-coqutil.0.0.5"
+        PACKAGES="${PACKAGES} coq-coqutil.0.0.6"
         PACKAGES="${PACKAGES} coq-rewriter.0.0.11"
         PACKAGES="${PACKAGES} coq-riscv.0.0.5"
-        PACKAGES="${PACKAGES} coq-bedrock2.0.0.7"
-        PACKAGES="${PACKAGES} coq-bedrock2-compiler.0.0.7"
-        PACKAGES="${PACKAGES} coq-rupicola.0.0.9"
+        PACKAGES="${PACKAGES} coq-bedrock2.0.0.8"
+        PACKAGES="${PACKAGES} coq-bedrock2-compiler.0.0.8"
+        PACKAGES="${PACKAGES} coq-rupicola.0.0.10"
         if [ "$OSTYPE" != cygwin ]
         then
           # Windows: stack overflow
```

Comment:
This is probably not the right place to discuss this (maybe open an issue on fiat crypto or here?), but fiat crypto does build with opam on Windows / cygwin, according to [our CI](https://github.com/mit-plv/fiat-crypto/actions/runs/8790991452/job/24124236671).